src/mixins/validation-mixin.ts

```diff
@@ -0,0 +1,95 @@
+import { Component, Vue } from 'vue-property-decorator'
+
+/**
+ * Mixin that provides some useful validation utilities.
+ */
+@Component({})
+export default class ValidationMixin extends Vue {
+  /**
+   * Creates a Vuetify rules object from the Vuelidate state.
+   * @param model The name of the model we are validating.
+   * @returns A Vuetify rules object.
+   */
+  public createVuetifyRulesObject (model: string): { [attr: string]: Array<any> } {
+    const obj = {
+      streetAddress: [],
+      streetAddressAdditional: [],
+      addressCity: [],
+      addressRegion: [],
+      postalCode: [],
+      addressCountry: [],
+      deliveryInstructions: []
+    }
+
+    // ensure Vuelidate state object is initialized
+    if (this.$v && this.$v[model]) {
+      // iterate over Vuelidate object properties
+      Object.keys(this.$v[model])
+        // only look at validation properties
+        .filter(prop => prop.charAt(0) !== '$')
+        .forEach(prop => {
+          // create array for each validation property
+          obj[prop] = []
+          // iterate over validation property params
+          Object.keys(this.$v[model][prop].$params)
+            .forEach(param => {
+              // add specified validation functions to array
+              switch (param) {
+                case 'required': obj[prop].push(() => this.requiredRule(model, prop)); break
+                case 'minLength': obj[prop].push(() => this.minLengthRule(model, prop)); break
+                case 'maxLength': obj[prop].push(() => this.maxLengthRule(model, prop)); break
+                case 'isCanada': obj[prop].push(() => this.isCanadaRule(model, prop)); break
+                case 'isBC': obj[prop].push(() => this.isBCRule(model, prop)); break
+                case 'validatePostalCode': obj[prop].push(() => this.postalCodeRule(model, prop)); break
```

Comment:
This was the main update here (and the local method on lines 92-94).

This code attaches to the model the validation functions that were provided in the schema, and provides a message if the validation fails. To me, it feels a bit weird that the validation is external but the error message is here, but that's how we originally got [Vuelidate](https://vuelidate.js.org/#sub-basic-form) working with the shared repo and somewhat flexible.